pyuvsim/source.py

```diff
@@ -40,6 +40,12 @@ def __init__(self, name, ra, dec, freq, stokes, pos_tol=np.finfo(float).eps):
                 4 element vector giving the source [I, Q, U, V]
             freq: astropy quantity
                 frequency of source catalog value
+            rise_lst: (float)
+                Approximate lst (radians) when the source rises. Set by coarse horizon cut in simsetup.
+                Default is None, meaning the source never rises.
+            set_lst: (float)
+                Approximate lst (radians) when the source sets.
+                Default is None, meaning the source never sets.
```

Comment:
It sounds to me like these may not be really optional parameters -- if you set neither of them what happens?

Reply:
ok, sounds reasonable.